src/Program.fs

```diff
@@ -759,7 +759,7 @@ let previewResolutionActions
 
     for library in librariesWithoutActions do
         for npmPackage in library.InteropDependencies do
-            logger.Information("{Library} requires npm package {Package}", library.Name, npmPackage.Name)
+            logger.Information("{Library} requires {PackageManager} package {Package}", library.Name, nodeManager.CommandName, npmPackage.Name)
```

Comment:
`nodeManager` should probably be renamed to `packageManager`

Reply:
fixed